libs/blocks/merch-card/merch-card.css

```diff
@@ -52,6 +57,10 @@ merch-card a[is="checkout-link"].upgrade:not(:first-of-type) {
     min-width: 66px; /* same as merch links */
     padding: 4px 18px 5px 21px;
     font-size: var(--consonant-merch-card-mini-compare-mobile-cta-font-size);
+  }
 
+  merch-card span[is="inline-price"].big-price {
```

Comment:
can we use the t-size naming like:  size-l, size-m, xxl etc?

Reply:
also how about a style 
`merch-card strong > em > span[is="inline-price"]` ? 

thus, you can avoid DOM manipulation maybe?

Reply:
fixed, now using l-size (following convention of CTAs size names) and using more specific querySelector `merch-card strong > em > span[is="inline-price"]`